account_asset_management/account_asset_invoice.py

```diff
@@ -4,7 +4,7 @@
 #    OpenERP, Open Source Management Solution
 #
 #    Copyright (C) 2010-2012 OpenERP s.a. (<http://openerp.com>).
-#    Copyright (c) 2014 Noviat nv/sa (www.noviat.com). All rights reserved.
+#    Copyright (c) 2014-now Noviat nv/sa (www.noviat.com).
```

Comment:
Now is a vague reference, because it depends on the moment of visualization. Please put an absolute reference (2015)